### DISCUSSION TITLE
Unable to add images once uploaded

### DISCUSSION THREAD

**Discussion by mlbiam** [requarks/wiki#6405] [Error / Bug Report] [answered]
Using wikijs 2.5.298 with MySQL, when I upload an image or add a folder to the assets window I get this error:

```
with recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFolders` where `id` = 0 union select `a`.`id`, `a`.`name`, `a`.`slug`, `a`.`parentId` from `assetFolders` as `a` inner join `ancestors` on `ancestors`.`parentId` = `a`.`id`) select * from `ancestors` - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1
```

Is there a configuration option I need to set?

**Comment by Dis9999** [comment]
We have the same error. Table `ancestors` doesn't exist in database, so this query returns an arror.

Details:

services:

  db:
    image: mysql:5.7.29
    container_name: wiki-db
    ...

   wiki:
    image: ghcr.io/requarks/wiki:2
    container_name: wiki
    depends_on:
      - db

Does anyone resolve the problem?

**Comment by NGPixel** [comment, marked as answer]
Read the [requirements](https://docs.requarks.io/install/requirements#database), MySQL 5.7 is only partially supported. Upgrade to a modern MySQL version.

**Reply by Dis9999** [reply]
Definitely sure. We had to switch to MySql 8.0. Thank you.
